dts/arm/renesas/smartbond/da1469x.dtsi

```diff
@@ -16,6 +16,15 @@
 		zephyr,flash-controller = &flash_controller;
 	};
 
+	power-states {
+		standby: standby {
+			compatible = "zephyr,power-state";
+			power-state-name = "standby";
+			min-residency-us = <500>;
+			exit-latency-us = <100>;
+		};
+	};
```

Comment:
power-states needs to be child of cpus node by convention

Reply:
moved according to convention